python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -600,8 +599,19 @@ def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList
         """
         self.log.debug("Computing coadd over %s", bbox)
         tempExpName = self.getTempExpDatasetName(self.warpType)
-        coaddMaskedImage = coaddExposure.getMaskedImage()
-        coaddMaskedImage.getMask().addMaskPlane("CLIPPED")
+        coaddExposure.mask.addMaskPlane("REJECTED")
+        coaddExposure.mask.addMaskPlane("CLIPPED")
+        coaddExposure.mask.addMaskPlane("SENSOR_EDGE")
+        # If a pixel is rejected due to a mask value other than EDGE, NO_DATA,
+        # or CLIPPED, set it to REJECTED on the coadd.
+        # If a pixel is rejected due to EDGE, set the coadd pixel to SENSOR_EDGE.
+        # if a pixel is rejected due to CLIPPED, set the coadd pixel to CLIPPED.
+        edge = afwImage.Mask.getPlaneBitMask("EDGE")
+        noData = afwImage.Mask.getPlaneBitMask("NO_DATA")
+        clipped = afwImage.Mask.getPlaneBitMask("CLIPPED")
+        toReject = statsCtrl.getAndMask() & (~noData) & (~edge) & (~clipped)
```

Comment:
Thanks for being so careful here. Without excluding those others, this feature wouldn't be useful.